version/version.go

```diff
@@ -23,9 +23,6 @@ func GetVersion() *VersionInfo {
 	if GitDescribe != "" {
 		ver = GitDescribe
 	}
-	if GitDescribe == "" && rel == "" && VersionPrerelease != "" {
-		rel = "dev"
-	}
```

Comment:
Why was this removed?

Reply:
Because it's unreachable. Right above it, rel is assigned the value of VersionPrerelease and then the condition requires rel to be an empty string and VersionPrerelease to be not empty.

Reply:
I've looked into this and the condition has been like this for 8 years. If no one noticed this in all this time, I really don't think that it's that important.